spec/factories/cfe_results/v6/mock_results.rb

```diff
@@ -1115,6 +1115,68 @@ def self.with_partner
         result
       end
 
+      # NOTE: this is a fake setup to exercise a CFE result with remarks that result in "all" review reasons
+      # Amend to add any new remarks as necessary.
+      def self.with_partner_and_employment_and_ineligible_remarks
```

Comment:
Would renaming it to `with_all_remarks` be clearer?

Reply:
certainly shorter 😄